README.md

```diff
@@ -1,8 +1,11 @@
 # Template for extending ASReview with new model
 
-ASReview has support for extensions, which enable you to seemlessly integrate
+<a href="https://github.com/asreview/asreview"><img src="https://img.shields.io/badge/ASReview-v1.0-%23ffcb05" alt="ASReview v1.0" /></a>
```

Comment:
Can you use markdown here? What is the added value btw of this badge? It works with 0.x versions as well.

Reply:
My main motivation here was that I like badges.

Reply:
I meant it to show that it's 1.0 compatible.